main.tf

```diff
@@ -395,3 +402,49 @@ resource "null_resource" "install_kube_vip_daemonset" {
   }
 }
 
+resource "null_resource" "worker_pre_reqs" {
```

Comment:
@displague @bikashrc25 : We need to have a `depends_on` pointing to this resources from `resource "null_resource" "deploy_anthos_cluster" `

This could cause a race condition since the deploy_anthos_cluster uses ansible under the covers to install packages such as docker via `apt`